Use Element: replaceChildren() method to remove all child nodes.

beru opened the pull request:
This PR replaces `innerHTML = ''` with `replaceChildren()` to remove all child nodes.

https://stackoverflow.com/a/65413839
https://developer.mozilla.org/en-US/docs/Web/API/Element/replaceChildren

`view.FindSidebar.deactivate` is also updated in the hope of resolving the problem reported in #1358. Although the change doesn't solve the problem, closing the sidebar gets faster.

<details><summary>recording</summary>

[efficientdet-d3.onnx](https://github.com/phantrdat/onnx-efficientdet/blob/5efe7b1eb45d69f7ca4eef1022540aa8a2024ae8/efficientdet-d3.onnx) file is used.

![lifespan](https://github.com/user-attachments/assets/0dfc800c-01ab-4f63-bb6a-cc046873ddf9)

</details>

lutzroeder commented:
[replaceChildren](https://caniuse.com/?search=replaceChildren) is more recent and requires Chrome 86. Is the improvement from calling in `deactivate` and would iterating on `lastChild` with `removeChild` work?

beru commented:
> [replaceChildren](https://caniuse.com/?search=replaceChildren) is more recent and requires Chrome 86.

Added a polyfill function for that. https://github.com/lutzroeder/netron/pull/1359/commits/e8b72a52531030cca85d229430a389e62adbb4ef

> Is the improvement from calling in deactivate and would iterating on lastChild with removeChild work?

Yes, I've confirmed that doing so still works.

I've found out that since I've added `this._content.replaceChildren();` line in `view.FindSidebar.deactivate`, I need to add `view.FindSidebar.activate` method and call `this._update();` in there. And the code in `view.FindSidebar.focus` needed to moved to `view.FindSidebar.activate`. I hope this change is correct. Please examine it.

-----

I've noticed that when I double-click Connections/Weights list item on FIND sidebar or click a connection on the graph view, CONNECTION PROPERTIES/TENSOR PROPERTIES sidebar is opened. After closing the sidebar, FIND sidebar reappers. But this is not the case with Nodes list item. After closing NODE PROPERTIES sidebar, FIND sidebar doesn't reappear. I thought this is a bit incosistent but I'm not sure what is a proper behavior here.
